<launch>
    <arg name="frequency" default="0.2"/>
    <arg name="imgPath" default=""/>
    <arg name="scaleFactor" default="5"/>
    <arg name="grayscale" default="false"/>
    <arg name="orig_x" default="0"/>
    <arg name="orig_y" default="0"/>
    <arg name="pixel_per_meter" default="500"/>

    <node name="img2rviz_publisher" pkg="img2rviz" type="img2rviz_node.py" output="screen">
        <param name="~frequency" value="$(arg frequency)" type="double"/>
        <param name="~imgPath" value="$(arg imgPath)" type="str"/>
        <param name="~scaleFactor" value="$(arg scaleFactor)" type="double"/>
        <param name="~grayscale" value="$(arg grayscale)" type="bool"/>
        <param name="~pixel_per_meter" value="$(arg pixel_per_meter)" type="double"/>
    </node>
    <node pkg="tf" type="static_transform_publisher" name="img2rviz" args="$(arg orig_x) $(arg orig_y) 0 0 0 0 /map /img2rviz  100"/>
</launch>
